expression/builtin_encryption_vec.go

```diff
@@ -43,6 +43,11 @@ func (b *builtinAesDecryptSig) vectorized() bool {
 
 func (b *builtinAesDecryptSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
 	n := input.NumRows()
+	if n == 0 {
```

Comment:
Maybe add a comment to explain why add check `n == 0`, because normally we handle `n == 0` in the `for loop` implicitly .